protostar-rust/src/pretty_printing.rs

```diff
@@ -0,0 +1,74 @@
+use anyhow::Error;
+use cairo_felt::Felt252;
+use camino::Utf8Path;
+use console::style;
+
+use cairo_lang_runner::RunResultValue;
+
+use super::TestsStats;
+
+pub fn print_error_message(error: &Error) {
+    let error_tag = style("ERROR").red();
+    println!("[{error_tag}] {error}");
+}
+
+pub fn print_collected_tests_count(tests_num: usize, tests_files_num: usize) {
+    let plain_text = format!("Collected {tests_num} test(s) and {tests_files_num} test file(s)");
+    println!("{}", style(plain_text).bold());
+}
+
+pub fn print_running_tests(test_file: &Utf8Path, tests_num: usize) {
+    let plain_text = format!("Running {tests_num} test(s) from {test_file}");
+    println!("{}", style(plain_text).bold());
+}
+
+pub fn print_test_summary(tests_stats: TestsStats) {
+    println!(
+        "{}: {} passed, {} failed ",
+        style("Tests").bold(),
+        tests_stats.passed,
+        tests_stats.failed
+    );
+}
+
+pub fn print_test_result(test_name: &str, result_value: &RunResultValue) {
+    let result_str = get_result_str(result_value);
+
+    let passed = get_if_passed(result_value);
+    let result_tag = if passed {
+        style("PASS").green()
+    } else {
+        style("FAIL").red()
+    };
+
+    let result_str_bold = style(result_str).bold();
+
+    println!("[{result_tag}] {test_name} {result_str_bold}");
+}
+
+fn get_result_str(run_result: &RunResultValue) -> String {
+    match run_result {
+        RunResultValue::Panic(result_data) | RunResultValue::Success(result_data) => {
+            result_data_to_text(result_data)
+        }
+    }
```

Comment:
Do we need match here at all?

Reply:
How else would you extract `result_data`?